stdpopsim/catalog/AraTha/genetic_maps.py

```diff
@@ -14,20 +22,14 @@
         populations. To get a single map for each chromosome, the Haldane map
         function distances were converted to recombination rates (cM/Mb) for
         each cross and then averaged across the 17 populations using loess.
+        The map was constructed on genome version TAIR7, and lifted to TAIR10.
+        The liftover methods used are available in the gzipped tarball in file
```

Comment:
```suggestion
        The liftover methods used are available in the gzipped tarball in the file
```

Reply:
Hm, but actually - the gzipped tarball is not really visible to the user (they'd have to do some digging to find the URL and get it themselves). How about just say it was lifted here, and put your notes in the documentation, at https://popsim-consortium.github.io/stdpopsim-docs/latest/development.html#lifting-over-a-genetic-map ?

Reply:
In those docs, no need to make a fully general solution/script like we had before (because we don't know if this is generalizable), just put what you've got with "here's what I had to do for AraTha..."